packages/gateway/test.eth.json

```
@@ -10,7 +10,8 @@
     "*.test.eth": {
         "addresses": {
             "60": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
-            "2":  "0xa914b48297bff5dadecc5f36145cec6a5f20d57c8f9b87"
+            "2":  "0xa914b48297bff5dadecc5f36145cec6a5f20d57c8f9b87",
+            "0": "0x007c96701e171f91d3a5a98477c4ead125add88425e4af98c1"
```

The above didn't work for me. Copy-pasting the address from the previous entry for `test.eth` did.

```suggestion
            "0": "0x76a91462e907b15cbf27d5425399ebf6f0fb50ebb88f1888ac"
```